app/routes/start/hello.js

```diff
@@ -4,6 +4,15 @@ import OnboardingRouteMixin from '../../mixins/onboarding-route';
 export default Ember.Route.extend(OnboardingRouteMixin, {
   currentUser: Ember.inject.service(),
 
+  actions: {
+    continueIfUserHasName() {
```

Comment:
I think this action should be renamed `attemptToContinue` or something similar. You're leaking implementation details from inside the action.

You want someone hitting enter to mean we try to continue given some set of circumstances, but those circumstances are really internal to the action itself.